types/create-test-server/index.d.ts

```diff
@@ -6,7 +6,9 @@ import { Express } from "express";
 import * as http from "http";
 import * as https from "https";
 
-type Server = createTestServer.TestServer & Omit<Express, "listen"> & { get: (url: string, response: string) => void };
+type Server = createTestServer.TestServer & Omit<Express, "listen"> & {
+    get: (url: string, response: string | (() => string)) => void;
```

Comment:
added `response: () => string)` — that type is explicitly present in the package test